packages/workbox-precaching/models/PrecachedDetailsModel.mjs

```diff
@@ -46,25 +48,25 @@ export default class PrecachedDetailsModel {
   async _getRevision(entryId) {
     const db = await this._getDb();
     const data = await db.get(entryId);
-    return data ? data.revision : null;
+    return data ? data[REVISON_FIELD] : null;
   }
 
   /**
    * Add an entry to the details model.
    * @param {PrecacheEntry} precacheEntry
    */
-  async addEntry(precacheEntry) {
+  async _addEntry(precacheEntry) {
     const db = await this._getDb();
     await db.put(precacheEntry._entryId, {
-      revision: precacheEntry._revision,
+      [REVISON_FIELD]: precacheEntry._revision,
```

Comment:
Is the switch to an array-keyed revision field necessary for the minification wins here?

Reply:
It's a small win, means 'revision' string previously entered twice or so can be replaced by a variable name and the variable name can be mangled to something like `a`.